app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardNetworksStep/MappingWizardNetworksStep.js

```diff
@@ -107,14 +108,25 @@ class MappingWizardNetworksStep extends React.Component {
 
     const { selectedCluster, selectedClusterMapping } = this.state;
 
+    const clusterOptions = getClusterOptions(clusterMappings);
+
     return (
       <div>
-        <SourceClusterSelect
```

Comment:
Can we delete the `SourceClusterSelect` component now @michaelkro @AparnaKarve ?

Reply:
Fo sho!  This is a huge upgrade, nice job @AparnaKarve!

Reply:
👍 alright sounds good. If we remove `SourceClusterSelect` I think this PR is good to merge 😄

Reply:
Thanks for the feedback guys -- will be deleting `SourceClusterSelect` now...